src/testportal/index.ts

```diff
@@ -70,15 +70,15 @@ export async function* getQuestions(
   await page.setDefaultNavigationTimeout(0);
   await page.goto(testURL.url.href, { waitUntil: 'networkidle0' });
   await fillForm(page);
-  await submitForm(page);
-  await page.waitForNavigation({ waitUntil: 'load' });
-
+  await Promise.all([
```

Comment:
are you sure that's required? Maybe there are some cases when submitForm Promise resolves after fully loaded next page, so `await page.waitForNavigation({ waitUntil: 'load' });` never resolves.

Reply:
nevermind, that looks okay, it may fail if `page.waitForNavigation` will execute before successful form submission and when page is still loading due to previous request here: `await page.goto(testURL.url.href, { waitUntil: 'networkidle0' });`(waitUntill sometimes doesn't work properly for unknown reasons). But I think that won't happen.